language/tools/test_generation/src/transitions.rs

```diff
@@ -142,7 +142,7 @@ pub fn stack_satisfies_struct_signature(
     for (i, token_view) in field_token_views.enumerate() {
         let abstract_value = AbstractValue {
             token: token_view.as_inner().clone(),
-            kind: token_view.kind(),
+            kind: token_view.kind(&vec![]).unwrap(),
```

Comment:
`SignatureTokenView::kind` will need to know the type formals of the immediate context (function/struct) the type is in, no matter we move the verification logic into the verifier or not, so this code will be broken. 

Adding proper support for generics to the tool will require a lot more effort, possibly refactoring AbstratState, so I'm making a simple fix here to preserve the original behavior. Does it look good to you guys, @shazqadeer & @darioncassel?

Reply:
Yes, that looks good to me for now. I'll work on updating the code to be aware of the context later. Thanks!